google/cloud/security/common/gcp_type/firewall_rule.py

```diff
@@ -17,10 +17,14 @@
 See: https://cloud.google.com/compute/docs/reference/latest/firewalls
 """
 
+import netaddr
```

Comment:
Your tests are failing because netaddr isn't included in the list of python modules to install in setup.py and scripts/docker/base